lib/net/http/header.rb

```diff
@@ -806,6 +806,107 @@ def set_form_data(params, sep = '&')
 
   alias form_data= set_form_data
 
+  # Stores form data to be included in the body of a +POST+ or +PUT+ request.
+  # When the request is sent, the data is URL-encoded
+  # (see {URI#encode_www_form}[https://docs.ruby-lang.org/en/master/URI.html#method-c-encode_www_form])
+  # and included in the request body.
+  #
+  # Note: There is no method to retrieve the form data directly;
+  # here we define a convenience method to do just that
+  # (its details are unimportant):
+  #
+  #   def get_form(req)
+  #     YAML.unsafe_load(req.to_yaml.lines[1..].join)['body_data']
+  #   end
+  #
+  # Any input referred to here as a _string_
+  # may actually be a
+  # {string-convertible object}[https://docs.ruby-lang.org/en/master/implicit_conversion_rdoc.html#label-String-Convertible+Objects].
```

Comment:
This is confusing to me. When I read the word "string", it's difficult for me to understand whether you mean string or a string-convertable object.

Reply:
Maybe better to just say 'string' and omit the note?

Reply:
Yeah I think that would be less confusing.

Reply:
Removed.